app/models/organisation_type.rb

```diff
@@ -6,7 +6,7 @@ class OrganisationType
     executive_agency:            { name: "Executive agency",                       analytics_prefix: "EA", agency_or_public_body: true,  non_departmental_public_body: false, allowed_promotional: false },
     executive_ndpb:              { name: "Executive non-departmental public body", analytics_prefix: "PB", agency_or_public_body: true,  non_departmental_public_body: true, allowed_promotional: false },
     advisory_ndpb:               { name: "Advisory non-departmental public body",  analytics_prefix: "PB", agency_or_public_body: true,  non_departmental_public_body: true, allowed_promotional: false },
-    tribunal_ndpb:               { name: "Tribunal non-departmental public body",  analytics_prefix: "PB", agency_or_public_body: true,  non_departmental_public_body: true, allowed_promotional: false },
+    tribunal:                    { name: "Tribunal",                               analytics_prefix: "PB", agency_or_public_body: true,  non_departmental_public_body: false, allowed_promotional: false },
```

Comment:
What are the implications of changing the `non_departmental_public_body` value to `false` here?

Reply:
I'm not sure. Do you reckon it would break anything?

Reply:
I think it's worth investigating further at least, as it would be good to avoid breaking anything (and that seems possible with this kind of change).

Reply:
The only place I found that would be affected by this change is this: https://github.com/alphagov/whitehall/blob/b54f7e2524d42679cda986fa8b552bca5aa3c1dd/app/views/admin/organisations/_form.html.erb#L123

Now I'm thinking maybe we should leave it as true?

Reply:
> Now I'm thinking maybe we should leave it as true?

Good investigating :+1: . That seems reasonable to me, otherwise those form elements would disappear from some organisations to which I guess they could be relevant.